src/main/java/com/flair/bi/web/websocket/FbEngineWebSocketService.java

```diff
@@ -39,6 +39,25 @@ public void pushGRPCMetaDeta(QueryResponse queryResponse, String request) {
         messagingTemplate.convertAndSendToUser(queryResponse.getUserId(), "/exchange/metaData", queryResponse.getData(), header);
     }
 
+    /**
+     * Send meta to users subscribed on channel "/user/exchange/metaData".
+     * <p>
+     * The message will be sent only to the user with the given username.
+     *
+     * @param queryResponse query response
+     * @param request request
+     */
+    public void pushGRPCMetaDataSharedLink(QueryResponse queryResponse, String request) {
```

Comment:
method name does not seem to be describing the method body. There seem to be nothing related to shared link in this method, so perhaps method name could be improved?